examples/nginx-basic-playbook/site.yml

```diff
@@ -10,7 +10,7 @@
       - python-devel
       - gcc
     nginx_test_message: This is a test message
-    nginx_keepalive_timeout: 115
+    nginx_webserver_port: 80
```

Comment:
This is fine but the smoke test task below starting on line 48 fails if you change this default value during your run. You need `url: http://localhost:{{ nginx_webserver_port }}/` there.